### PR TITLE
Add new RoleBindings to broker manual setup

### PR DESCRIPTION
**Pull request by grantr**
These were added to the automatic setup in https://github.com/knative/eventing/pull/1426. They're not required for the Broker to work, but they enable observability features like consolidated logging, metrics, and tracing.

## Proposed Changes

- Document additional RoleBindings to be created when manually preparing a namespace for a Broker.

/cc @Fredy-Z

### CODE REVIEW

File: docs/eventing/broker-trigger.md [hunk 1]
```diff
@@ -124,16 +124,19 @@ required once per namespace. These instructions will use the `default`
...
 ```shell
 kubectl -n default create serviceaccount eventing-broker-ingress
 kubectl -n default create serviceaccount eventing-broker-filter
 ```
 
-Then give it the needed RBAC permissions:
+Then give them the needed RBAC permissions:
 
 ```shell
+kubectl -n default create rolebinding eventing-broker-ingress \
```
Comment (Fredy-Z) [inline docs/eventing/broker-trigger.md]: 137-139 are exactly the same as 140-142, added by mistake?

Comment (grantr) [reply docs/eventing/broker-trigger.md]: Good catch! This command was added by mistake.

**Comment by samodell** [comment]
Nice, @grantr ! And thanks for the review, @Fredy-Z !

/approve

**Pull request merged**